js/scripts.js

```diff
@@ -0,0 +1,16 @@
+var a = '2',
+    b = '4',
+    value = (a*a) + (2 * a * b) - (b*b)
+;
+console.log('Wynik: '+value+' ');
+
+
+if (value > 0) {
+    console.log('wynik dodatni');
+    
+}   else if (value < 0) {
+    console.log('wynik ujemny');
+} 
+
+var równyZero = value = 0 ? 'Wynik jest równy zero' : 'Wynik nie jest równy zero';
```

Comment:
po co tutaj oddzielny warunek? wystarczy w 13 linijce dorzucić jeszcze jednego `else` bo już w 13 linijce masz pewność, że liczba nie jest ani dodatnia ani ujemna, więc wystarczy wyświetlić, że jest równa zero na `elsie`

jeśli chciałbyś do tego wykorzystać ternary to można to by było zrobić tak:
```
if (value > 0) {
    console.log('dodatni');
} else {
    value < 0 ? console.log('ujemny') : console.log('rowny zero');
}
```